deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -58,6 +58,7 @@ master:
   # be removed with it in a future release
   port: 8080
   metricsPort: 8081
+  healthPort: 8082
```

Comment:
im ok with this being general health port while we map to grpc-health flag. later on it can map to a different flag if/once needed.